### PR TITLE
Contrib Inventory: NSoT

### PR DESCRIPTION
**Pull request by coxley**
## Network Source of Truth

Flexible IPAM/CMDB by Dropbox, I feel like this is a good feature to have upstream. Didn't notice any other similar choices, ignoring cloud inventory providers because this enables network engineers to have a better source of truth for their devices.

Full README is located over at [coxley/ansible_nsot](https://github.com/coxley/ansible_nsot) where I have some integration tests for parameters and a basic HTTP mock. I've tried to make it as easy for the user to take advantage of this.

### CODE REVIEW

File: contrib/inventory/nsot.py [hunk 1]
```diff
@@ -0,0 +1,345 @@
+#!/bin/env python2.7
```
Comment (bcoca) [inline contrib/inventory/nsot.py]: you might want to make it python vs python2.7 which will not work on all platforms

Comment (coxley) [inline contrib/inventory/nsot.py]: Done

**Comment by coxley** [comment]
Are there any other issues here? :)

**Comment by coxley** [comment]
This has been running in production for months at our place reliably. If there are any concerns, just let me know.

**Comment by bcoca** [comment]
sorry for the delay, i was hoping someone from the community would review and tests this, I did not see anything that would prevent us from merging it so doing it now.

**Pull request merged**